index.html

```diff
@@ -1,184 +1,352 @@
+            <a href="#" class="connect">Let's Connect</a>
+            <div class="social-icon">
+              <ul>
+                <li>
+                  <a href="#"
+                    ><img src="/img/twiter icon.svg" alt="Twitter icon"
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"
+                    ><img src="/img/Linkedin icon.svg" alt="LinkedIn Icon"
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"
+                    ><img src="/img/medium icon.svg" alt="medium icon"
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"
+                    ><img src="/img/github icon.svg" alt="github icon"
+                  /></a>
+                </li>
+                <li>
+                  <a href="#"><img src="/img/hi icon.svg" alt="Hi icon" /></a>
+                </li>
+              </ul>
             </div>
-        </section>    
+            <button class="btn btn-primary">Get My Resume</button>
+          </div>
+          <div class="skils">
+            <ul class="skills-items">
+              <li class="skill-item">
+                <h3 class="lang">
+                  Languages <img src="/img/down arrow.png" alt="" />
+                </h3>
+                <br />
+                <span class="item">
+                  <img src="/img/js.png" alt="" />
+                  <h4>Javascript</h4></span
+                >
+                <br />
+                <span class="item">
+                  <img src="/img/html.png" alt="" />
+                  <h4>HTML</h4></span
+                >
+                <br />
+                <span class="item">
+                  <img src="/img/css.png" alt="" />
+                  <h4>CSS</h4></span
+                >
+                <br />
+              </li>
+              <li class="skill-item">
+                <h3 class="lang">
+                  Frameworks <img src="/img/Union.png" alt="" />
+                </h3>
+              </li>
+              <hr />
+              <li class="skill-item">
+                <h3 class="lang">Skills <img src="/img/Union.png" alt="" /></h3>
+              </li>
+              <hr />
+            </ul>
+          </div>
+        </div>
+      </section>
+
+      <section id="contact-section">
+        <div class="contact-container">
+          <div class="contact-decs">
+            <h2>Contact me</h2>
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+          </div>
+          <div class="contact-form">
+            <form
+              id="fs-frm"
+              name="simple-contact-form"
+              accept-charset="utf-8"
+              action="https://formspree.io/f/{{form-id}}"
+              method="post"
```

Comment:
- I get an error when I try to submit the form, and that is because you have an invalid formspree id. Please use a valid formspree id. This is necessary so users/recruiters can reach out to you through your portfolio. Please ensure you fix this 😄 😄